packages/kbn-build/src/utils/packageJson.js

```diff
@@ -0,0 +1,6 @@
+import readPkg from 'read-pkg';
+import path from 'path';
+
+export function readPackageJson(dir, depName = '') {
```

Comment:
__question:__ what is `depName`? In what case will we need this parameter?

Reply:
Ah, just stuff left over from earlier explorations. Removing it.